packages/vx-demo/src/components/VxDocLink.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Link from 'next/link';
+
+function getDocUrlFromVXPackageName(vxPackage: string) {
+  const packageName = vxPackage.split('@vx/')[1]; // e.g., @vx/shape
+  return packageName ? `docs/${packageName}` : null;
+}
+
+type Props = {
+  packageName: string;
+};
+
+const VxDocLink = ({ packageName }: Props) => {
+  const url = getDocUrlFromVXPackageName(packageName);
+  return url ? <Link href={url}>{packageName}</Link> : null;
+};
+
+export default VxDocLink;
```

Comment:
![gif](https://media.giphy.com/media/146heXDX89mUgw/giphy.gif)

**but** the vx casing here hurts me